ADMINS-FOR-ISTIO.md

```diff
@@ -6,21 +6,21 @@ access.
 
 | Service | Purpose | Responsible Adults
 |---------|---------|-------------------
-| [CircleCi](https://circleci.com) | Secondary CI system | [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
-| [Discourse](https://discuss.istio.io) | Discussion forums | [askmeegs](https://github.com/askmeegs), [linsun](https://github.com/linsun), [oaktowner](https://github.com/oaktowner), [rshriram](https://github.com/rshriram)
-| [Docker Hub](https://hub.docker.com) | Container registry | [rshriram](https://github.com/rshriram), [costinm](https://github.com/costinm), [mandarjog](https://github.com/mandarjog), [vadimeisenbergibm](https://github.com/vadimeisenbergibm), [frankbu](https://github.com/frankbu)
-| [GitHub](https://github.com/istio) | Source control, bug tracking | [howardjohn](https://github.com/howardjohn), [rshriram](https://github.com/rshriram), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [CircleCi](https://circleci.com) | Secondary CI system | [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [Discourse](https://discuss.istio.io) | Discussion forums | [askmeegs](https://github.com/askmeegs), [linsun](https://github.com/linsun), [craigbox](https://github.com/craigbox), [rshriram](https://github.com/rshriram)
+| [Docker Hub](https://hub.docker.com) | Container registry | [brian-avery](https://github.com/brian-avery), [costinm](https://github.com/costinm), [mandarjog](https://github.com/mandarjog), [vadimeisenbergibm](https://github.com/vadimeisenbergibm), [frankbu](https://github.com/frankbu)
```

Comment:
I think @howardjohn is a Docker Hub admin already. I'm currently an admin for the istio-ecosystem Docker Hub, but not istio itself.

Reply:
Your reward is to make yourself an admin :)